Pipeline/PackageBuildOutputs/README.md

```diff
@@ -289,47 +294,77 @@ ArtifactoryHelpers console output
+
   -verb,--verbose                                Flag to provide more log output. (Optional)
+
   -il,--includeLogs                              Comma-separated list of files/patterns
                                                  from the USS build workspace
+
   -ae,--addExtension                             Flag to add the deploy type extension to the member
                                                  in the package tar file. (Optional)                                                                                              
-                                        
-  Optional Artifactory Upload opts:
+
+  -h,--help                                      Prints this message
+
+  Optional Artifact Repsository upload opts:
  
-  -p,--publish                                   Flag to indicate package upload to
-                                                 the provided Artifactory server.
-                                                 (Optional)
-  -artifactory,
-    --artifactoryPropertiesFile <propertyFile>   Absolute path of a property file
-                                                 containing application specific
-                                                 Artifactory details. (Optional)
-  -v,--versionName <versionName>                 Name of the Artifactory version.
-                                                 (Optional)
+  -p,--publish
+                     Flag to indicate package upload to
+                     the provided Artifactory server.
+                     (Optional)
  
+  -v,--versionName <versionName>                 
+                     Name of the Artifactory version. (Optional)
+
+  -ad,--artifactRepositoryDirectory <repoDirectory>
+                     Directory path in the repository to store the build . (Optional)
  
-  -h,--help                                      Prints this message
+  -aprop,--artifactRepositoryPropertyFile <propertyFile>
+                     Path of a property file containing application specific artifact
+                     repository details. (Optional)
+
+  -ar,--artifactRepositoryName <repoName>
+                     Artifact repository name to store the build. (Optional)
+ 
+  -au,--artifactRepositoryUrl <url>
+                     URL to the Artifact repository server. (Optional)
+
+  -aU,--artifactRepositoryUser <user>
+                     User to connect to the Artifact repository server. (Optional)
+  
+  -aP,--artifactRepositoryPassword <password>
+                     Password to connect to the Artifact repository server. (Optional)
+
+ -artifactory,--artifact repositoryPropertiesFile <Artifactory
+ repositoryPropertiesFile>  
+                     Path of a property file containing
+                     application specific Artifactory repository details. (Optional) ** (Deprecated)
+ 
+
 ```
 
-## Command Line Options Summary - ArtifactoryHelpers
+## Command Line Options Summary - ArtifactRepositoryHelpers
```

Comment:
It could be good to mention that for
Nexus -> repo type must be raw
Artifactory -> repo type must be generic
I had a situation where someone tested this script with a Nexus maven repo. Result is unpredictable.